mne/io/tests/test_raw.py

```diff
@@ -582,3 +583,66 @@ def test_describe_df():
                               -9.643554807784935e-13,
                               1.928710961556987e-12,
                               4.146728567347522e-11]))
+
+
+def test_get_data_units():
+    """Test the "units" argument of get_data method."""
+    # Test the unit conversion function
+    assert _get_scaling('eeg', 'uV') == 1e6
+    assert _get_scaling('grad', 'fT/cm') == 1e13
+    assert _get_scaling('csd', 'uV/cm²') == 1e2
```

Comment:
Hi @agramfort, I put the conversion code in a new private function and added a few lines to test it, is that alright?

Reply:
can you improve a bit with eg:

```
    assert _get_scaling('eeg', 'uV') == 1e6
    assert _get_scaling('eeg', 'dV') == 1e1
    assert _get_scaling('eeg', 'pV') == 1e12
    assert _get_scaling('mag', 'fT') == 1e15
    assert _get_scaling('grad', 'T/m') == 1
    assert _get_scaling('grad', 'T/mm') == 1e-3
    assert _get_scaling('grad', 'fT/m') == 1e15
    assert _get_scaling('grad', 'fT/cm') == 1e13
    assert _get_scaling('csd', 'uV/cm²') == 1e2
    ```

Reply:
>     assert _get_scaling('eeg', 'dV') == 1e1

`dV`?? Have you been drinking? 😅😊 Is this actually being used? 😱